http_examples.cpp

```diff
@@ -142,7 +164,7 @@ int main() {
       // Check if path is within web_root_path
       if(distance(web_root_path.begin(), web_root_path.end()) > distance(path.begin(), path.end()) ||
          !equal(web_root_path.begin(), web_root_path.end(), path.begin()))
-        throw invalid_argument("path must be within root path");
+        throw invalid_argument("Path must be within root path");
```

Comment:
Exception messages typically starts with lower case

Reply:
Hmm
I made them upper case to match the std::cerr messages that start with upper case letters, but did not know they should be lower case. Revert this please.
Thanks for the tip